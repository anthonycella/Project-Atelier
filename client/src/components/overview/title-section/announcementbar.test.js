import renderer from 'react-test-renderer';
import AnnouncementBar from './announcementbar.jsx';

it('Announcement bar renders to the page', () => {
  const component = renderer.create(
    <AnnouncementBar />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
