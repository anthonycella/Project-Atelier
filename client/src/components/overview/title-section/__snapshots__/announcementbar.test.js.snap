// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Announcement bar renders to the page 1`] = `
<div
  className="sc-bczRLJ bnzveL"
>
  SITE-WIDE ANNOUNCEMENT MESSAGE! SALE/DISCOUNT 
  <strong>
    OFFER
  </strong>
   - 
  <u>
    NEW PRODUCT HIGHLIGHT
  </u>
</div>
`;
